templates/zerver/help/search-for-messages.md

```diff
@@ -1,104 +1,150 @@
 # Searching for messages
 
-Zulip has a powerful search engine under its hood. Search for messages using
-the search bar at the top of the app.
+Zulip has a powerful search engine under its hood. You can search for messages using
```

Comment:
> Zulip has a powerful search engine under its hood.

We might want to consider rewriting this phrase in a follow-up pass; it looks like we wrote that back in 2018 (6880af6c2e3b4e598f385e3c2efb2eba4e2ce226).